modules/sql-engine/src/test/java/org/apache/ignite/internal/sql/engine/framework/TestBuilders.java

```diff
@@ -138,6 +138,8 @@ public interface TableBuilder extends TableBuilderBase<TableBuilder> {
         /** Returns a builder of the test hash-index object. */
         public HashIndexBuilder hashIndex();
 
+        TableBuilder colocationGroup(ColocationGroup colocationGroup);
```

Comment:
why do we need colocation group in TableBuilder? I mean, TableBuilder is meant to build table for planner tests, they don't need colocation group to be provided